fundamentals/conclusion.tex

```diff
@@ -16,6 +16,12 @@ \section{Conclusions}
 %Open paradigm allows community oversight, testing, verification
 %Enables apples to apples comparisons
 
+% gonuke: I am sensitive to making the implication that VISION (or others) has
```

Comment:
I've actually recently taken uncertain provenance out. I'm not sure what that was about.